search-commons/src/main/java/no/unit/nva/search2/model/ResourceParameterKey.java

```diff
@@ -36,25 +38,23 @@ public enum ResourceParameterKey implements ParameterKey {
     INVALID(STRING),
     // Parameters converted to Lucene query
     CATEGORY(STRING, "entityDescription.reference.publicationInstance.type"),
-    CONTRIBUTOR(STRING, "entityDescription.contributors.identity.id"
```

Comment:
no more search by contributor id?

Reply:
I need time to investigate this, it is really strange